src/library/Uno.Material/MaterialResourcesV2.cs

```diff
@@ -112,7 +112,9 @@ private IEnumerable<(string ResourceKey, string SharedKey, bool IsDefaultStyle)>
 			Add("MaterialOutlinedPasswordBoxStyle");
 			Add("MaterialOutlinedTextBoxStyle");
 			Add("MaterialProgressBarStyle", isImplicit: true);
+#if !WinUI
```

Comment:
I think it would be also great to have a bit more context in the code as well by adding a comment @kazo0